charts/telepresence/templates/deployment.yaml

```diff
@@ -80,7 +80,7 @@ spec:
           - name: INTERCEPT_DISABLE_GLOBAL
             value: {{ quote (default .intercept.disableGlobal false) }}
```

Comment:
I think this one is written wrong too and only works because its set in values.yaml (clearly the one you took as an example), can you fix it since you're at it and add it to the changelog ? :D

Reply:
@knlambert done!!!